test/test_utils.js

```diff
@@ -74,6 +74,12 @@ describe("Testing utils functions", function() {
             assert.equal(utils.readable_msisdn("0821234567", "+27"), "0821234567");
             assert.equal(utils.readable_msisdn("+27821234567", "27"), "0821234567");
         });
+        // the next test case should not occur as the msisdn's passed in should
+        // either be normalized (start with '+') or as is (start with '0')
+        it("should return msisdn as is if contains no leading +", function() {
+            assert.equal(utils.readable_msisdn("27821234567", "+27"), "27821234567");
+            assert.equal(utils.readable_msisdn("27821234567", "27"), "27821234567");
```

Comment:
it's open for discussion.  If the purpose of 'readable_msisdn' is to mainly be an inverse of the function that normalizes msisdn's, then we shouldn't ever get a msisdn passed in to 'readable_msisdn' that doesn't start with '+'.  Don't you agree?  But we might want to be able to deal with msisdn's like the one in question... What's your call; should I leave as is for now or improve?